roles/tendrl-alerting/tasks/source.yml

```diff
@@ -0,0 +1,51 @@
+#
+# Installation
+#
+
+- name: Directory for git repository
+  file:
+    path="{{ tendrl_alerting_repo }}"
+    state=directory
+
+- name: Clone git repository
+  git:
+    repo={{ tendrl_alerting_repo_url }}
+    version={{ tendrl_alerting_repo_branch }}
+    dest={{ tendrl_alerting_repo }}
+
+- name: Installation from source code
+  pip:
+    name=.
+    chdir={{ tendrl_alerting_repo }}
+
+- name: Use example config files as initial configuration (see installation.rst)
+  copy:
+    src={{ item.src }}
+    dest=/etc/tendrl/{{ item.dest }}
+    remote_src=True
+  with_items:
+    - src: "{{ tendrl_alerting_repo }}/etc/tendrl/tendrl.conf.sample"
+      dest: 'tendrl.conf'
```

Comment:
Are we sure we would like to do this? The `tendrl.conf` file is shared across components and so this step would remove any previous chagnes.